hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/WorldLedgers.java

```diff
@@ -133,6 +146,82 @@ public boolean defaultKycStatus(final TokenID tokenId) {
                 tokenId, ACC_KYC_GRANTED_BY_DEFAULT, StaticEntityAccess::defaultKycStatus);
     }
 
+    public Optional<TokenInfo> infoForToken(final TokenID tokenId, final ByteString ledgerId) {
+        if (staticEntityAccess != null) {
+            return staticEntityAccess.infoForToken(tokenId);
+        } else {
+            try {
+                final var token = tokensLedger.getImmutableRef(tokenId);
+                if (token == null) {
+                    return Optional.empty();
+                }
+                return token.asTokenInfo(tokenId, ledgerId);
+            } catch (Exception unexpected) {
```

Comment:
Seems fine, but afaict it _would_ be quite unexpected if an exception was thrown here.

Was there a repeatable failing EET that led you to add a `catch` here?

Reply:
I believe it was a left-over from a code copy from `StateView`. I removed the try/catch block.